src/crypto/ocb_internal.cc

```diff
@@ -496,49 +496,44 @@ static void ecb_decrypt_blks(block *blks, unsigned nblks, KEY *key) {
 #elif USE_NETTLE_AES
 /*-------------------*/
 
-// TODO(Issue 1202): Stop using deprecated nettle APIs
-#pragma GCC diagnostic push
-#pragma GCC diagnostic ignored "-Wdeprecated-declarations"
 #include <nettle/aes.h>
 
 namespace ocb_aes {
 
-typedef struct aes_ctx KEY;
+typedef struct aes128_ctx KEY;
 
 static KEY *KEY_new() { return new KEY; }
 
 static void KEY_delete(KEY *key) { delete key; }
 
-static void set_encrypt_key(const unsigned char *handle, const int bits, KEY *key)
+static void set_encrypt_key(const unsigned char *handle, const int /*bits*/, KEY *key)
```

Comment:
Now that we’re no longer using `bits`, maybe an `assert(bits == 128)` would be useful?

Reply:
Done